### DISCUSSION TITLE
t-navbar控件在iPhone全屏模式下(非浏览器，而是另存到桌面），位置有Bug

### DISCUSSION THREAD

**Discussion by smartweb** [Tencent/tdesign-mobile-vue#1533] [Q&A]
vue mobile版本，t-navbar控件在iPhone全屏模式下(非浏览器，而是另存到桌面），位置有Bug，直接顶到屏幕最上方，没有在安全边界内。加多以下两行css即可修复。

.t-navbar--fixed .t-navbar__content {
    top: constant(safe-area-inset-top); 
    top: env(safe-area-inset-top);
  }

**Comment by anlyyao** [comment]
@smartweb 欢迎直接提 pr 到仓库，欢迎一起共建🏹
